larsim/LegacyLArG4/ISCalculationCorrelated.cxx

```diff
@@ -131,7 +133,7 @@ namespace larg4 {
     // apply the scintillation pre-scaling (normally this is already folded into
     // the particle-specific scintillation yields)
     fNumScintPhotons *= fScintPreScale;
-
+    
```

Comment:
I don't understand why the code checks did not complain.  There is a bit of whitespace added on an empty line.

Reply:
This was just a typo -- I had added a "cout" statement around that line while testing the change, and I guess some whitespace stuck around when I removed those lines. Didn't think it would matter. Should I remove it?

Reply:
Yes please.

Reply:
Ok, done!